examples/vs.mk

```diff
@@ -18,7 +18,7 @@
 #    USA
 
 
-CFLAGS=/Iutils /I..\lib /MTd /DWIN32=1 
+CFLAGS=/Iutils /I..\lib /MTd /DWIN32=1 /W3
```

Comment:
equivalent to GCC's -Wall, more specifically /W3 is supposed to be "production-level" warnings